### DISCUSSION TITLE
Get parameters too long for +server.ts

### DISCUSSION THREAD

**Discussion by dalton5** [etherCorps/sveltekit-og#46] [Q&A]
Hi,

My svelte component take parameters a picture to add it as background. Then I pass image as x64. But I need to push it as get parameter to +server.ts. The issue is that the parameter is too long for get request.

Then I use POST, but I do not know how to deal with og meta.

Any idea?

**Comment by theetherGit** [comment]
OG Meta tags uses GET so you should use get method for it.
for image you can use 
- Upload functionality to upload image to cloud storage something like that and you can pass image endpoint as param.
- If you have static images and you have to choose from a list of images you can add them in `static` folder and then you can use file name as parameter and in OG route you can handle image route with your domain.
